neon_nodes/voice_client.py

```diff
@@ -61,19 +62,19 @@ def __init__(self, bus=None, ready_hook=on_ready, error_hook=on_error,
         self._node_config = self.config.get('neon_node', {})
         self._hana_address = self._node_config.get('hana_address')
 
-        LOG.init(self.config.get("logging"))
+        init_log(self.config, "neon-node")
         self.bus = bus or FakeBus()
         self.lang = self.config.get('lang') or "en-us"
-        self._mic = OVOSMicrophoneFactory.create(self.config)
+        self._mic = OVOSMicrophoneFactory.create(self.config['microphone'])
```

Comment:
Consider a defensive `.get()` instead of indexing, although this basically guarantees at a semi-helpful exception log

Reply:
I'm not sure which would be more helpful, either this logs a `KeyError` or the factory will (as of now) log a `TypeError` when it tries to load a plugin named `None`.

Maybe an additional check here with a more helpful log for empty configuration would be best..